src/app/navigation/selectors.ts

```diff
@@ -38,4 +40,19 @@ export const match = createSelector(
   (state) => state.match
 );
 
+export const systemQueryParameters = createSelector(
+  query,
+  (navQuery) => pickBy(isDefined, {
+    'content-style': navQuery['content-style'],
```

Comment:
when I included `navQuery.archive` here, it ended up duplicated in the query string on navigation